### PR TITLE
Fix BIND configs for modern distros

### PR DESCRIPTION
**Pull request by iliajie**
This is it. Jamie, have a look, please.

### CODE REVIEW

**Comment by iliajie** [comment]
Will `bind8/config-debian-linux-10.0-ALL` cover Ubuntus?

**Comment by iliajie** [comment]
I would also prefer to change to `systemctl` commands for other main modules used with Virtualmin?

File: bind8/config-AlmaLinux-6.0-ALL [hunk 1]
```diff
@@ -42,15 +40,13 @@ keygen=dnssec-keygen
 signzone=dnssec-signzone
 checkconf=named-checkconf
 other_slaves=1
-restart_cmd=restart
 keygen=dnssec-keygen
 tmpl_dnssec=0
 dnssec_period=21
 tmpl_dnssec_dt=1
-dnssectools_conf=/etc/dnssec-tools/dnssec-tools.conf
```
Comment (jcameron) [inline bind8/config-AlmaLinux-6.0-ALL:50]: Why remove these lines?

Comment (iliajie) [reply bind8/config-AlmaLinux-6.0-ALL:50]: I assumed that this no longer relevant nowadays. Isn't it?

Comment (jcameron) [reply bind8/config-AlmaLinux-6.0-ALL:50]: Maybe, but let's fix one  thing at a time in this PR. I think the main issue is using the incorrect stop command?

Comment (iliajie) [reply bind8/config-AlmaLinux-6.0-ALL:50]: > but let's fix one thing at a time in this PR.

Duh, thanks for reminding me, Jamie! I'm about to get used to fixing one issue at a time!

Comment (iliajie) [reply bind8/config-AlmaLinux-6.0-ALL]: > I think the main issue is using the incorrect stop command?

Yes, _stop_, _start_ and _reload_ command for new distros should use `systemctl` command.

**Comment by jcameron** [comment]
Thanks for the fix!

**Comment by iliajie** [comment]
Do you want me to add familiar changes for all other major modules we use with Virtualmin?

**Comment by jcameron** [comment]
Yes, as long as we can be sure that it's applied only for distro versions that use `systemd`

**Pull request merged**